.travis.yml

```diff
@@ -138,6 +138,7 @@ after_success:
 cache:
   directories:
     - "travis_phantomjs"
+    - $HOME/.composer/cache
```

Comment:
wouldn't it be better to only cache `$HOME/.composer/cache/files`, as everything else might change to often, causing cache invalidations?

Reply:
Maybe you're right.
I contribute to many open-source projects - like WP-CLI - and we use this directory.

Reply:
Change frequency equals how many times *we* change the lock file.

Reply:
Guess our lock file isn't changed that frequently

Reply:
Yes https://github.com/matomo-org/matomo/commits/3.x-dev/composer.lock